lib/archethic_web/controllers/faucet_controller.ex

```diff
@@ -40,22 +40,22 @@ defmodule ArchEthicWeb.FaucetController do
   def create_transfer(conn, %{"address" => address}) do
     with {:ok, recipient_address} <- Base.decode16(address, case: :mixed),
          true <- Crypto.valid_hash?(recipient_address),
-         :ok <- transfer(recipient_address) do
+         {tx_address, :ok} <- transfer(recipient_address) do
```

Comment:
The convention in Elixir/Erlang is to use `{:ok, result} ` where `result` is the data returned by the function. But the `ok` atom is specified in the beginning.